core/src/main/java/apoc/trigger/TriggerHandler.java

```diff
@@ -302,28 +209,11 @@ public void stop() {
         }
     }
 
-    private <T> T withSystemDb(Function<Transaction, T> action) {
-        try (Transaction tx = apocConfig.getSystemDb().beginTx()) {
-            T result = action.apply(tx);
-            tx.commit();
-            return result;
-        }
-    }
-
     private long getLastUpdate() {
         return withSystemDb( tx -> {
             Node node = tx.findNode(SystemLabels.ApocTriggerMeta, SystemPropertyKeys.database.name(), db.databaseName());
             return node == null ? 0L : (long) node.getProperty(SystemPropertyKeys.lastUpdated.name());
         });
     }
 
-    private void setLastUpdate(Transaction tx) {
-        Node node = tx.findNode(SystemLabels.ApocTriggerMeta, SystemPropertyKeys.database.name(), db.databaseName());
-        if (node == null) {
-            node = tx.createNode(SystemLabels.ApocTriggerMeta);
-            node.setProperty(SystemPropertyKeys.database.name(), db.databaseName());
-        }
-        node.setProperty(SystemPropertyKeys.lastUpdated.name(), System.currentTimeMillis());
-    }
```

Comment:
This is not right. If you get rid of this, then you don't need to do any scheduler job. Basically the condition:
```
if (getLastUpdate() > lastUpdate) {
    updateCache();
}
```

is never going to be true

Reply:
I moved this piece of code in `TriggerUtils`, method `setLastUpdate(String databaseName, Transaction tx)`